swagger.json

```diff
@@ -5118,6 +5229,83 @@
+                    "example": 1234
+                },
+                "vehicleId": {
+                    "type": "integer",
+                    "description": "Vehicle ID",
+                    "example": 212014918086169
+                },
+                "statusType": {
+                    "type": "string",
+                    "description": "The log status for this event.",
+                    "enum": [
+                        "OFF_DUTY", 
+                        "SLEEPER_BED", 
+                        "DRIVING",
+                        "AUTO_DUTY",
+                        "ON_DUTY", 
+                        "YARD_MOVE", 
+                        "PERSONAL_CONVEYANCE"
+                    ],
+                    "example": "OFF_DUTY"
+                },
+                "statusTypeCode": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "A numerical identifier for the status type with 0 corresponding to the first entry in the above \"statusType\" list.",
+                    "example": 1
+                },
+                "action": {
+                    "type": "string",
+                    "description": "The audited log change.",
+                    "enum": [
+                        "Created",
+                        "Deleted",
+                        "Updated",
+                        "Created (Auto Duty)",
+                        "Deleted (Auto Duty)",
+                        "Vehicle Assignment Created",
+                        "Vehicle Assignment Deleted",
+                        "Vehicle Assignment Updated",
+                        "Signed In",
+                        "Signed Out",
+                        "Pending Vehicle Assignment Created",
+                        "Pending Vehicle Assignment Deleted",
+                        "Pending Vehicle Assignment Updated",
+                        "Driver Signout",
+                        "Force Sign Out",
+                        "Cascade Signout",
+                        "Vehicle Driver Changed"
+                    ],
+                    "example": "Created"
+                },
+                "actionCode": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "A numerical identifier for the action with 0 corresponding to the first entry in the above \"action\" list.",
+                    "example": 1
+                },
+                "logStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "The time at which the log/HOS started in Unix epoch milliseconds.",
+                    "example": 1462881998034
+                },
+                "createdAtMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "The time in Unix epoch milliseconds that the audit entry was created.",
```

Comment:
Should we add some additional context describing the difference between logStartMs and createdAtMs? Maybe we could describe that if the vehicle gateway loses cellular connectivity for 5 minutes, createdAtMs will be 5 minutes behind logStartMs?

Reply:
sure!